docs/contributing.md

```diff
@@ -78,6 +77,16 @@ even though it doesn't fail on low coverage.
 
 Unit tests are automatically run on Travis CI for pull requests.
 
+### Advanced
+
+To run the complete set of unit tests run in CI for your platform, run the following
+in the repository root:
+
+```bash
+ci/setup_env.sh
+ACTIVATE_VENV=1 ci/test_all.sh
+```
+
```

Comment:
Here are the main commands you'd need to follow assuming a sane Python installation.